app/models/in_person_enrollment.rb

```diff
@@ -157,6 +157,10 @@ def enhanced_ipp?
 
   private
 
+  def days_to_expire
+    enhanced_ipp? ? IdentityConfig.store.in_person_eipp_enrollment_validity_in_days.days : IdentityConfig.store.in_person_enrollment_validity_in_days.days
```

Comment:
This'll probably fail Rubocop lint, but generally I'd feel this would read better and be more conventional as an `if` / `else`.

```suggestion
    if enhanced_ipp?
      IdentityConfig.store.in_person_eipp_enrollment_validity_in_days.days
    else
      IdentityConfig.store.in_person_enrollment_validity_in_days.days
    end
```